account_analytic_distribution/models/account_move_line.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# Copyright 2017 Vicent Cubells - <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    analytic_distribution_id = fields.Many2one(
+        comodel_name='account.analytic.distribution',
+        string='Analytic distribution',
+    )
+
+    def _analytic_line_distributed_prepare(self, line, rule):
+        res = self._prepare_analytic_line(line)
```

Comment:
I'm guessing you changed this method from an implicit api.multi to an implicit api.model because this is what happened in Odoo itself with _prepare_analytic_line. I have a slight preference to minimize this change as follows.
```
@api.multi
def _analytic_line_distributed_prepare(self, rule):
        res = self._prepare_analytic_line(self)
```

Reply:
Pushing this change myself @PieterPaulussen because if nothing else it fixes the last travis error: a line length error precisely because of the signature change.